pages/1.13/release-notes/1.13.0/index.md

```diff
@@ -593,31 +591,39 @@ To add this job definition to the JSON editor, you would modify the existing JSO
     * `--preflight`
     * `--install-prereqs`
     * `--deploy`
-    * `--post-flight`
+    * `--postflight`
+
+    If you attempt to use an option that is no longer valid, the installation script displays a warning message. You can also identify deprecated options by running the `dcos_generate_config.sh` script with the `--help` option. The output for the `--help` option displays [DEPRECATED] for the options that are no longer used. 
+    
+    These options will be removed in DC/OS 1.14. If you have scripts or programs that use any of the deprecated options, you should update them (DCOS-50263).
+
+- The CLI command `dcos node` has been replaced by the new command `dcos node list`. 
 
-    If you have scripts or programs that use any of the deprecated options, you should update them (DCOS-50263).
+    Running the `dcos node` command after installing this release automatically redirects to the output of the `dcos node list` command. The `dcos node list` command provides information similar to the output from the `dcos node` command, but also includes an additional column that indicates the public IP address of each node.
 
-- The CLI command `dcos node` has been replaced by the new command `dcos node list`. Running the `dcos node` command after installing this release automatically redirects to the output of the `dcos node list` command. The `dcos node list` command provides the same output plus includes an additional column that indicates the public IP address of each node.
+    If you have scripts or programs that use output from the `dcos node` command, you should test the output provided by the `dcos node list` command and the update your scripts or programs, as needed.
```

Comment:
Use `then` instead of `the` in this area: `the update your scripts or programs`.

Reply:
fixed.